### DISCUSSION TITLE
Style of call to fit

### DISCUSSION THREAD

**Discussion by dmbates** [JuliaStats/MixedModels.jl#586] [Show and tell]
To prevent long calls of the form
```julia
m4 = fit(MixedModel, @formula(y ~ 1 + a + b*c + x + (1+a|f) + ...), dat; contrasts)
```
where the formula tends to push all the other arguments off to the right, I have taken to writing a `let` block of the form
```julia
m4 = let
    f = @formula(y ~ 1 + a + b*c + x + (1+a|f) + ...)
    fit(MixedModel, f, dat; contrasts)
end
```
I also tend to create a constrasts specification called `contrasts` so that only the name of the argument needs to be specified.

By the way, many thanks to the authors of the `StandardizedPredictors` package, which has helped to make things much more understandable.

**Comment by palday** [comment]
> By the way, many thanks to the authors of the StandardizedPredictors package, which has helped to make things much more understandable.

@kleinschmidt and I are thrilled to see it used. :smile:
